gemfiles/rack_3.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rack", "~> 3.0", git: "https://github.com/rack/rack.git"
```

Comment:
Not sure you need the `~> 3.0`, but maybe you just delete the `git` option after the Rack 3 official release?

Reply:
Yes, that's correct.

Reply:
> but maybe you just delete the `git` option after the Rack 3 official release?

Done.